data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -88,6 +90,9 @@
                     [/gold_carryover]
 
                     [note]
+                        [show_if]
+                            {VARIABLE_CONDITIONAL moved_too_close boolean_equals false}
+                        [/show_if]
```

Comment:
I'm pretty sure this will have no effect, since any `[show_if]` is evaluated only when the `[objective]` tag is executed and not every time the objectives are shown. (It's not very intuitive, I'll admit, and perhaps it should be changed.)